PowerPointLabs/Test/FunctionalTest/ColorsLabTest.cs

```diff
@@ -289,7 +316,7 @@ private void DragAndDrop(Point startPt, Point endPt)
 
         private static void AssertEqual(List<Color> exp, List<Color> actual)
         {
-            for (int i = 0; i < 13; i++)
+            for (int i = 0; i < exp.Count; i++)
```

Comment:
Is exp `expected`? Is there a reason for shortening when `actual` is spelled in full?

Reply:
yes it is expected. No reason for shortening, I could see how it could cause some confusion... Changed! thanks.